integrations/gulp-postcss/gulpfile.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require('gulp');
+const postcss = require("gulp-postcss");
+const autoprefixer = require("autoprefixer");
```

Comment:
i would avoid using postcss, autoprefixer and postcss-calc dependencies for the purpose of these tests